src/docker-osx-dev

```diff
@@ -226,9 +225,9 @@ function do_rsync {
 #
 function sync {
   local readonly paths_to_sync=("$@")
-
-  for path in "${paths_to_sync[@]}"; do
-    do_rsync "$path"
+  for pathRsync in ${paths_to_sync[@]}
```

Comment:
`for path in ${paths_to_sync[@]}
   `
will output this : 
`
[DEBUG] rsync --archive --verbose --delete --omit-dir-times --inplace --whole-file --exclude .git --rsh="ssh -i /Users/hamza/.ssh/id_boot2docker -o StrictHostKeyChecking=no" /Users/hamza/projet/typo3conf docker@dockerhost:/Users/hamza/projet/



[DEBUG] rsync --archive --verbose --delete --omit-dir-times --inplace --whole-file --exclude .git --rsh="ssh -i /Users/hamza/.ssh/id_boot2docker -o StrictHostKeyChecking=no" /Users/hamza/projet/uploads docker@dockerhost:/Users/hamza/projet/



[DEBUG] rsync --archive --verbose --delete --omit-dir-times --inplace --whole-file --exclude .git --rsh="ssh -i /Users/hamza/.ssh/id_boot2docker -o StrictHostKeyChecking=no" /Users/hamza/projet/fileadmin docker@dockerhost:/Users/hamza/projet/
`
  `for path in "${paths_to_sync[@]}"
   `
And this will will output this 

[DEBUG] rsync --archive --verbose --delete --omit-dir-times --inplace --whole-file --exclude .git --rsh="ssh -i /Users/hamza/.ssh/id_boot2docker -o StrictHostKeyChecking=no" /Users/hamza/projet/typo3conf docker@dockerhost:/Users/hamza/projet/